HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -507,6 +507,10 @@ Choose an element from the collection at random.
 code::
 [1, 2, 3, 4].choose;
 ::
+There's an optional argument for specifying relative weights with a list. The length of both lists must be equal.
+code::
+[1, 2, 3, 4].choose([1, 2, 3, 4]);
```

Comment:
this example may be a bit confusing for beginners.
Perhaps something like this?
```
[110, 220, 330, 440].choose([1, 3, 1, 2]);
```

Reply:
Certainly!